typescript/expCalc.ts

```diff
@@ -0,0 +1,81 @@
+function Add(x,y)
```

Comment:
Please add function return and parameter types:
function Add(x: number, y: number): number